### ISSUE TITLE
Default favicon not loading.

### ISSUE THREAD

**Issue by mlonn** [Asymmetrik/node-fhir-server-core#173]
### If you are reporting a bug?

**What is the current behavior?**
Trying to start new server in 2.0.0 without a custom favicon. But the server does not start. 
Gives the following error,
```
Error: ENOENT: no such file or directory, stat '/Users/krulla/dev/fhir-node/src/assets/phoenix.ico'
    at Object.statSync (fs.js:848:3)
    at resolveSync (/Users/krulla/dev/fhir-node/node_modules/serve-favicon/index.js:187:17)
    at favicon (/Users/krulla/dev/fhir-node/node_modules/serve-favicon/index.js:61:12)
    at Server.configureMiddleware (/Users/krulla/dev/fhir-node/node_modules/@asymmetrik/node-fhir-server-core/src/server/server.js:146:16)
    at initialize (/Users/krulla/dev/fhir-node/node_modules/@asymmetrik/node-fhir-server-core/src/index.js:42:5)
    at Object.initialize (/Users/krulla/dev/fhir-node/src/index.js:15:14)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Module._compile (/Users/krulla/dev/fhir-node/node_modules/pirates/lib/index.js:99:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Object.newLoader [as .js] (/Users/krulla/dev/fhir-node/node_modules/pirates/lib/index.js:104:7)
```

**What is the expected behavior?**

Server should start.
**What are the steps to reproduce?**
Follow the get started guid in the docs.
**What OS are you using and what version of node.js and @asymmetrik/node-fhir-server-core are you running?**

OS: macOS
node version: 10.12.0
@asymmetrik/node-fhir-server-core: 2.0.0

I could get the server to start by adding the icon to `/src/assets/phoenix.ico`

**Comment by Robert-W** [comment]
Ahh thanks, good catch. the path being interpreted was relative to the user directory and not the node module. Pushing a patch now

**Comment by Robert-W** [comment]
Ok, we published a patch for this already, 2.0.1 is available !!
